### ISSUE TITLE
Cant install, what programs do i need installed?

### ISSUE THREAD

**Issue by Jphgamer14** [werwolf2303/SpotifyXP#41]
What programs do i need installed? I cant get the program installed from a fresh xp install with java installed

**Comment by ciberfan** [comment]
> What programs do i need installed? I cant get the program installed from a fresh xp install with java installed

hi there, let me help. First of all, ¿are you using windows XP SP3?, what version of java are you using? do yoy already compile the java executable?, ¿what version are you using of java?

**Comment by Jphgamer14** [comment]
Yes I am using Xp sp3, I am using java version 153 and the I use the jar file from the releases tab.

**Comment by Jphgamer14** [comment]
Let me rephrase, I can install it I just cant open it

**Comment by werwolf2303** [comment]
Do you still have the problem?

**Comment by ciberfan** [comment]
> Let me rephrase, I can install it I just cant open it

![image](https://github.com/werwolf2303/SpotifyXP/assets/143445988/45af86cb-80b1-4100-964c-508fffc32d33)
hello, sorry for the delay in replying, I have had a health problem. Ok, let's get to the point.
Bearing in mind that you have installed and registered the java runtime environment, you must install the vlc 3.0.18 program, and then create a new text file with notepad and change the extension from .txt to .bat containing the following: 
java -jar SpotifyXP.jar --setup-complete
must go together in the same directory.

**Comment by werwolf2303** [comment]
It should create a Desktop shortcut with the necessary arguments? Does this not work for you?

**Comment by Jphgamer14** [comment]
> > Let me rephrase, I can install it I just cant open it
> 
> ![image](https://user-images.githubusercontent.com/143445988/271049102-45af86cb-80b1-4100-964c-508fffc32d33.png) hello, sorry for the delay in replying, I have had a health problem. Ok, let's get to the point. Bearing in mind that you have installed and registered the java runtime environment, you must install the vlc 3.0.18 program, and then create a new text file with notepad and change the extension from .txt to .bat containing the following: java -jar SpotifyXP.jar --setup-complete must go together in the same directory.

Thank you, after install vlc it started working
